src/common/useI18n.ts

```diff
@@ -2,9 +2,15 @@ import { useConfig } from './useConfig';
 import { formatDate } from '../i18n/date';
 
 export function useI18n(): { formatDate: (date: Date) => string } {
-  const { defaultLanguage } = useConfig();
+  const config = useConfig();
+
+  if (config) {
+    return {
+      formatDate: (date) => formatDate(date, config.defaultLanguage),
+    };
+  }
```

Comment:
Couldn't we just always pass `config.defaultLanguage`, as the function will then use the default argument?